dispatch/apps/content/models.py

```diff
@@ -455,13 +473,14 @@ def save(self, **kwargs):
             image = Img.open(StringIO.StringIO(self.img.read()))
             self.width, self.height = image.size
             super(Image, self).save()
-            name = re.split('.(jpg|gif|png)', self.img.name)[0]
+            name = re.split('.(jpg|JPEG|jpeg|JPG|gif|png)', self.img.name)[0]
+            fileType = re.split('.(jpg|JPEG|jpeg|JPG|gif|png)', self.img.name)[1]
```

Comment:
Define `"jpg|JPEG|jpeg|JPG|gif|png"` as a constant somewhere to make it easier to add more filetypes in the future